src/pages/[platform]/build-a-backend/data/aws-appsync-apollo-extensions/index.mdx

```diff
@@ -223,15 +203,145 @@ val authorizer = IamAuthorizer {
 //highlight-end
 ```
 
+</Block>
+
+<Block name="Without Amplify">
```

Comment:
I hesitate with putting prose like this inside the block switcher for a few reasons:
1. it makes linking to specific content difficult
2. nesting headings messes with the document structure -- do they render okay in the TOC?
3. since there are multiple instances of the blockswitcher and they do not sync on the page it can make reading this doc "without amplify" a bit tedious

We can modify the component to fix 3, but I have concerns with 2 if the heading structure is different between the two blocks